Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,10 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    var [elementWidth, elementHeight] = (this.pressInLocation)
+    ? [this.pressInLocation.pageX - e.nativeEvent.locationX, this.pressInLocation.pageY - e.nativeEvent.locationY]
+    : [0, 0];
+    this._dispatchHotspotUpdate(e.nativeEvent.pageX - elementWidth, e.nativeEvent.pageY - elementHeight);
```

Comment:
**this.pressInLocation** is set in Touchables.js. _performSideEffectsForTransition() calls _savePressInLocation() if its a new highlight. Since this doesn't get updated during an active touch i used its absolute pageX/pageY for calculation.
**e.nativeEvent** does always contain the new absolute pageX/pageY values and the relative locationX/locationY vals. Here the relative values stay the same.

I didn't found an easier way to calculate the current relative position than this but i also didnt want to go too deep.
At the moment its (current absolute X - (static relative X - static absolute X)).